### ISSUE TITLE
Allow reverse suggestion ordering

### ISSUE THREAD

**Issue by r-bar** [nvim-lua/completion-nvim#259]
**Is your feature request related to a problem? Please describe.**
Vim's default completion menu behavior is frustrating. When the completion menu appears the most relevant suggestion is placed at the bottom of the list, furthest from the cursor. This means that your eye must find the best suggestion furthest away in a (potentially rapidly changing) list.

**Describe the solution you'd like**
It would be helpful to provide a way to reverse the order of the items in the completion menu.

**Comment by Shougo** [comment]
I think the example screenshot is needed.

**Comment by jemag** [comment]
I agree that this can be quite frustrating especially with bigger projects with a lot of dependencies. This is an example from a java project:

![image](https://user-images.githubusercontent.com/7985687/104131135-8b015780-5342-11eb-9b89-455ecde32b11.png)

Notice that there is lookupOperations just above, but if I start typing "loo", I will not even see lookupOperations suggested, despite it being the most relevant. To see that suggestion, I would have to scroll way down:
![image](https://user-images.githubusercontent.com/7985687/104131040-04e51100-5342-11eb-89a8-2316bb2b4227.png)

This means that you either have to scroll a lot and search through a ton of results, or type a really significant portion of the word, which kind of defeats the purpose of having a completion engine.
